assets/javascripts/src/components/Main.jsx

```diff
@@ -48,12 +49,24 @@ function mapDispatchToProps(dispatch) {
     return {
         goBack: () => dispatch({ type: GO_BACK }),
         goForward: () => dispatch({ type: GO_FORWARD }),
```

Comment:
as we talked about I don't think it's necessary as making it compatible for mobile covers the "next" button only displayed on desktops